README.md

```diff
@@ -1,9 +1,12 @@
 # zsh-pyenv-lazy
 
-A zsh plugin for lazy loading of pyenv.
+A zsh plugin for installing and lazy loading of pyenv.
+
+`pyenv` will be installed if `$HOME/.pyenv` is empty.
```

Comment:
> if `$HOME/.pyenv` is empty

That's not exactly true if being strict, right? Currently it tries to install if it's not a directory.

Reply:
See my other comment about this below

Reply:
Sorry for replying too late. I have refined it, please have a review.